### PR TITLE
Added dev CORS policy and modified route id gen

### PR DESCRIPTION
**Pull request by maffe03**
Added a CORS policy to allow for localhost development. Also changed route operation id to be less wordy and duplicated which will make codegen much nicer. :)

### CODE REVIEW

File: main.py [hunk 1]
```diff
@@ -22,12 +28,27 @@ async def lifespan(app: FastAPI):
...
+]
+
 app = FastAPI(
     lifespan=lifespan,
     redoc_url=None if no_docs else "/redoc",
     docs_url=None if no_docs else "/docs",
+    generate_unique_id_function=generate_unique_id,
 )
 
+if environ.get("ENV") == "development":
```
Comment (Equasa) [inline main.py:43]: byt ut environ mot getenv()

File: main.py [hunk 2]
```diff
@@ -1,12 +1,18 @@
...
 from os import environ
 
 
+def generate_unique_id(route: APIRoute):
+    return f"{route.name}"
+
+
 @asynccontextmanager
 async def lifespan(app: FastAPI):
     if environ.get("ENV") == "development":
```
Comment (Equasa) [inline main.py:18]: byt till os.getenv()

**Review by Equasa — CHANGES_REQUESTED** [review]
looks good, just need to change from environ

**Comment by PelicanQ** [comment]
Cool! This does however not guarantee unique operationId's. It takes the names of our functions.
Here I have one route in root router and one in user router. both function name `some_name`. In openapi.json, both have `some_name` as operationId. 
image
image

Is there a point to having nice short operationId's even if you won't be writing them explicitly?

**Comment by maffe03** [comment]
Yes you're right, OpenAPI requires operationIds to be unique. The latest commit should address this. It might not be the case for routes without tags, but I hope we add tags to all our routes anyways. (We currently have 4 test routes in main.py without tags but the rest do. Could probably add a prefix like 'base' otherwise?). 

With this commit it should be possible to do good, almost not janky, codegen on the frontend, while making sure ids (under different routers) are unique. I don't know if FastAPI has any protection against routes within a router having the same name. Does python forbid this? 

Atleast this is (almost) identical to the [guide](https://fastapi.tiangolo.com/advanced/generate-clients/#custom-generate-unique-id-function) on FastAPI.

**Review by Equasa — APPROVED** [review]

**Pull request merged**